etl-driver/src/main/java/com/wipro/ats/bdre/im/etl/api/BaseLoad.java

```diff
@@ -72,16 +68,14 @@ private void processStage(String dbName, String baseTableName, String instanceEx
 
             List<String> stagePartitions = new ArrayList<String>();
             ResultSet resultSet = getHiveJDBCConnection(dbName).createStatement().executeQuery("show partitions " + stageTableName);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "instanceExecId" is provided externally to the method and not sanitized before use. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS2077)